src/Illuminate/Database/Eloquent/SoftDeletes.php

```diff
@@ -38,7 +38,7 @@ public function forceDelete()
     /**
      * Perform the actual delete query on this model instance.
      *
-     * @return void
+     * @return mixed
```

Comment:
void is meant to indicate the returned value should not be relied upon from the outside, so not sure this is a correct change